### PR TITLE
User profile revamp - KB - 4301, 4151, 4084 is fixed. KB - 2568

### PR DESCRIPTION
**Pull request by vShyave**

### CODE REVIEW

File: project/ws/app/src/lib/routes/profile-v2/components/withdraw-request/withdraw-request.component.html [hunk 1]
```diff
@@ -2,9 +2,14 @@
     <div class="flex">
         <mat-icon class="info-icon">info_outline</mat-icon>
     </div>
-    <!-- <div class="flex flex-wrap text-base primary">
-        You requested for transfer with following details:
-    </div> -->
+    <div class="flex flex-col text-sm primary">
+        <div>You requested for transfer with the following details:</div>
```
Comment (vishnubansaltarento) [inline project/ws/app/src/lib/routes/profile-v2/components/withdraw-request/withdraw-request.component.html:6]: Multi-lingual for same required

File: project/ws/app/src/lib/routes/profile-v2/routes/profile-view/profile-view.component.ts [hunk 2]
```diff
@@ -44,7 +44,7 @@ export const MY_FORMATS = {
   },
 }
 const EMAIL_PATTERN = /^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,4}$/
-const MOBILE_PATTERN = /^\d{10}$/
+const MOBILE_PATTERN = /^[0]?[6789]\d{9}$/
```
Comment (vishnubansaltarento) [inline project/ws/app/src/lib/routes/profile-v2/routes/profile-view/profile-view.component.ts:47]: This validation to be apply in MDO as well?

File: src/app/home/home/notification/notification.component.html [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<ng-container *ngIf="data.type === 'pending'">
+    <div class="flex flex-row gap-8 items-center">
+        <div class="flex-1">
+            Still the request is pending, click <span class="primary cursor-pointer" (click)="handleRoute()" >here</span> to check profile page
```
Comment (vishnubansaltarento) [inline src/app/home/home/notification/notification.component.html:4]: Multi Lingual text required?

File: project/ws/app/src/lib/routes/profile-v2/routes/profile-view/profile-view.component.ts (hunk 2 above)
Comment (vShyave) [reply project/ws/app/src/lib/routes/profile-v2/routes/profile-view/profile-view.component.ts:47]: Yes! Same applied too

File: src/app/home/home/notification/notification.component.html (hunk 3 above)
Comment (vShyave) [reply src/app/home/home/notification/notification.component.html:4]: Still, some more UI implementation is pending, once it is done. Then I will add

**Pull request merged**